mapalign/embed.py

```diff
@@ -362,21 +372,16 @@ def _get_affinity_matrix(self, X, Y=None):
                     # currently only symmetric affinity_matrix supported
                     self.affinity_matrix_ = 0.5 * (self.affinity_matrix_ +
                                                    self.affinity_matrix_.T)
-                    return self.affinity_matrix_
             if self.affinity == 'rbf':
                 self.gamma_ = (self.gamma
                                if self.gamma is not None else 1.0 / X.shape[1])
                 self.affinity_matrix_ = rbf_kernel(X, gamma=self.gamma_)
-                return self.affinity_matrix_
             if self.affinity in ['markov', 'cauchy']:
                 from .dist import compute_affinity
                 self.affinity_matrix_ = compute_affinity(X,
                                                          method=self.affinity,
                                                          eps=self.gamma,
                                                          metric=self.metric)
-                return self.affinity_matrix_
-            self.affinity_matrix_ = self.affinity(X)
-            return self.affinity_matrix_
```

Comment:
the function description should be modified if this function is not going to return the affinity matrix.

Reply:
I re-added the return of affinity_matrix_, but not as a field in self. The reason is that in the code I am currently writing I want to be able to bind results of several affinity calculations to different fields in the object.